ckanext/lacounts/harvest/processors/ckan.py

```diff
@@ -1,21 +1,32 @@
+import logging
 from urlparse import urljoin
 from ckanext.lacounts.harvest import helpers
+log = logging.getLogger(__name__)
 
 
 def ckan_processor(package, harvest_object):
     source_domain = helpers.extract_source_domain(harvest_object)
 
-    # all sources
+    # general
     package['harvest_dataset_url'] = '{harvest_source_url}/dataset/{id}'.format(**package)
+    package['contact_name'] = package.get('contact_name')
+    package['contact_email'] = package.get('contact_email')
+    package['temporal_text'] = package.get('temporal_coverage')
+    package['temporal_start'] = None
+    package['temporal_end'] = None
 
     # data.chhs.ca.gov
     if source_domain == 'data.chhs.ca.gov':
```

Comment:
Rather than have special cases for each site why don't we try to abstract the logic to work across all of them. Something like this:

```python

mappings = {
  'spatial_text': ['geo_coverage', 'spatial_coverage'],
  'frequency': ['accrual_periodicity'],
  'provenance': ['author', 'program],
}

for key, field_names in mappings:
   if not key in package:
       for field_name in field_names:
             if package.get(field_name):
                  package[key] = package[field_name]
             # TODO: also check package extras with the same field names
    
```

This way the only thing we need to add when harvesting new sites is new potential field names to the mapping (if necessary)

Reply:
@amercader 
If we expect new datasets have been added mapping makes sense. For 2 data sources, it felt like an overkill. I'll tweak it.